akka-http-core/src/main/scala/akka/http/scaladsl/model/ErrorInfo.scala

```diff
@@ -47,6 +47,8 @@ final class ErrorInfo(
 
   /** INTERNAL API */
   @InternalApi private[akka] def this(summary: String, detail: String) = this(summary, detail, "")
+
+  override def toString(): String = s"ErrorInfo($summary, $detail, $errorHeaderName)"
```

Comment:
Do we want the detail in the `toString`? I'd say yes, but people might think otherwise, given, that the feature is there to prevent accidental leakage of information. Of course, leakage to the client is still prevented but details might leak to logs.